### ISSUE TITLE
Upgrade to latest version is failing

### ISSUE THREAD

**Issue by sks444** [mbraak/django-file-form#422]
We just upgraded our django-file-form version to 3.1.2. And the ci is failing with following message:

```python
Migrations for 'django_file_form':
  venv/lib/python3.7/site-packages/django_file_form/migrations/0007_auto_20210118_1009.py
    - Create model TemporaryUploadedFile
    - Delete model UploadedFile
```
It looks like the existing model(UploadedFile) was renamed to TemporaryUploadedFile in https://github.com/mbraak/django-file-form/pull/394 but the migrations file for same was not added.

**Comment by mbraak** [comment]
I will have a look

**Comment by sks444** [comment]
Sure.

There usually is a migrations file with [RenameModel](https://docs.djangoproject.com/en/3.1/ref/migration-operations/#renamemodel) operation in this case.

**Comment by mbraak** [comment]
Ok, thanks. I will add the migration.

**Comment by mbraak** [comment]
Fixed in version 3.1.3

**Comment by sks444** [comment]
Thanks @mbraak for the quick release!
